e2e/tests/ui/features/@search/search.feature

```diff
@@ -0,0 +1,71 @@
+Feature: Search
+	As a Devsecops Engineer
+	I want to perform searching across vulnerabilities, SBOMs and packages, specific searches for CVE IDs, SBOM titles, package names and show results that are easy to navigate to the specific item of interest.
+
+Background:
+	Given User is authenticated
+	And User is on the Search page
+
+Scenario: User visits search page without filling anything
+	Then a total number of 17 "SBOMs" should be visible in the tab
+	And a total number of 5537 "Packages" should be visible in the tab
+	And a total number of 29 "Vulnerabilities" should be visible in the tab
+	And a total number of 57 "Advisories" should be visible in the tab
+
+Scenario Outline: User toggles the "<types>"list and manipulates the list
+ 	When User selects the Tab "<types>" 
+	Then the "<types>" list should have specific filter set
+	And the "<types>" list should be sortable
```

Comment:
We should probably make sure each column is sortable. BUT I think that this may be redundant, as this is already being tested [here](https://github.com/guacsec/trustify-ui/tree/main/e2e/tests/ui/pages/sbom-list).

Reply:
Though on second thought it may be a good idea to still test this, as BDD scenarios are likely to run separately from the tests written by Carlos, but the test step could re-use the methods defined in his page objects.